src/linear_algebra/determinant.lean

```diff
@@ -40,7 +40,8 @@ open equiv equiv.perm finset function
 namespace matrix
 open_locale matrix big_operators
 
-variables {n : Type u} [decidable_eq n] [fintype n] {R : Type v} [comm_ring R]
+variables {m n : Type u} [decidable_eq n] [fintype n] [decidable_eq m] [fintype m] {R : Type v}
```

Comment:
Are you sure that m and m have to be from the same universe? (I haven't checked how matrix is defined)

Reply:
Looks like it would work if `perm_on_inr_of_perm_on_inl` is also changed accordingly.